src/main/java/at/hannibal2/skyhanni/utils/TimeUtils.kt

```diff
@@ -177,7 +184,7 @@ enum class TimeUnit(val factor: Long, val shortName: String, val longName: Strin
     fun format(value: Int, longFormat: Boolean = false) = value.addSeparators() + getName(value, longFormat)
 }
 
-val Duration.inPartialSeconds: Double get() = inWholeMilliseconds.toDouble() / 1000
+val Duration.inPartialSeconds: Double get() = toDouble(DurationUnit.SECONDS)
```

Comment:
have you tested that `to double seconds` is the same as `in whole milliseconds to double div 1k`?